.github/workflows/integration_tests.yml

```diff
@@ -16,21 +16,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        keptn-version: [ "0.12.7", "0.13.5", "0.14.2", "0.15.0" ] # https://github.com/keptn/keptn/releases
+        keptn-version: ["0.12.7", "0.13.5", "0.14.2", "0.15.0"] # https://github.com/keptn/keptn/releases
     env:
-      GO_VERSION: 1.17
       GO111MODULE: "on"
       ENABLE_E2E_TEST: true
       BRANCH: ${{ github.head_ref || github.ref_name }}
       KUBECONFIG: /etc/rancher/k3s/k3s.yaml
     steps:
       - name: Install Go
-        uses: actions/setup-go@v3.0.0
+        uses: actions/setup-go@v3.1.0
         with:
-          go-version: ${{ env.GO_VERSION }}
+          go-version-file: "go.mod"
 
       - name: Checkout keptn-contrib/keptn-service-template-go
-        uses: actions/checkout@v3
+        uses: actions/checkout@v3.0.2
```

Comment:
Here we still need to move the checkout before setup-go

Reply:
Fixed